tests/conftest.py

```diff
@@ -157,6 +157,7 @@ def a_volume_node(a_slicer_app, a_nrrd_volume_file_path):
 def a_segmentation_node(a_segmentation_editor, a_volume_node):
     segmentation_node = a_segmentation_editor.create_empty_segmentation_node()
     a_segmentation_editor.set_active_segmentation(segmentation_node, a_volume_node)
+    a_segmentation_editor.add_empty_segment()
```

Comment:
It's better to leave the default segmentation node empty. This way individual tests can decide how they want to populate the segmentation for their usecase.